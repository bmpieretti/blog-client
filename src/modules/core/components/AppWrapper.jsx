import React from 'react';
import ErrorBoundary from './ErrorBoundary';
import GlobalStyles from './GlobalStyles';
import App from './App';

export default () => (
  <React.Fragment>
    <ErrorBoundary>
      <App />
    </ErrorBoundary>
    <GlobalStyles />
  </React.Fragment>
);
