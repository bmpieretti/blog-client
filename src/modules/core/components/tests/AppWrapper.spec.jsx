import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AppWrapper from '../AppWrapper';

describe('App Component: ', () => {
  let wrapper;
  beforeEach(() => { wrapper = shallow(<AppWrapper />); });

  test('should render wrappers', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
