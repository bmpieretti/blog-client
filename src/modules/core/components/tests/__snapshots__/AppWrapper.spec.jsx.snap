// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component:  should render wrappers 1`] = `
<Fragment>
  <Styled(ErrorBoundary)>
    <Styled(App) />
  </Styled(ErrorBoundary)>
  <GlobalStyleComponent />
</Fragment>
`;
